media/css/firefox/browsers-products.scss

```diff
@@ -549,6 +549,10 @@ $url-download-link-hover: svg-url($download-link-hover);
             right: $layout-md;
```

Comment:
You can remove the existing properties for both `left` and `right` now that the mixin is defining them.

Nit: try and keep things in alphabetical order, so the `@include bidi()` can come first at the top of the list.

Thanks!